packages/gatsby/src/bootstrap/load-plugins/validate.js

```diff
@@ -4,7 +4,7 @@ const stringSimiliarity = require(`string-similarity`)
 const { version: gatsbyVersion } = require(`gatsby/package.json`)
 const reporter = require(`gatsby-cli/lib/reporter`)
 const resolveModuleExports = require(`../resolve-module-exports`)
-const getLatestAPIs = require(`../../utils/get-latest-apis`)
+const { getLatestAPIs } = require(`../../utils/get-latest-apis`)
```

Comment:
```suggestion
import { getLatestAPIs } from "../../utils/get-latest-apis"
```

Reply:
not necassary as these files are still js. When they converted they will switch to import

Reply:
Yeah, I think so too.